src/query/ast/tests/it/testdata/stmt-error.txt

```diff
@@ -160,7 +160,7 @@ error:
   --> SQL:1:6
   |
 1 | show database
-  |      ^^^^^^^^ unexpected `database`. Did you mean `SHOW DATABASES`?
+  |      ^^^^^^^^ unexpected `database`. this is a reserved keyword, please avoid using it. Did you mean `SHOW DATABASES`?
```

Comment:
In this case should not print `this is reserve...`

Reply:
ok